src/application.ts

```diff
@@ -32,6 +35,7 @@ export default class Application implements Owner {
   public rootElement: any;
   public resolver: Resolver;
   public env: Environment;
+  protected roots: object[] = [];
```

Comment:
Yes let's do that. It should be `Option<Simple.Node>` because `nextSibling` is nullable